src/common/Types.ts

```diff
@@ -449,6 +450,26 @@ export enum AccessibilityTrait {
     None
 }
 
+// When multiple components with autoFocus=true are mounting at the same time,
+// and/or multiple focus() calls are happening during the same render cycle,
+// it is possible to specify a callback which will choose one from those multiple.
+// To set this callback use View's arbitrateFocus property.
+export type FocusArbitrator = (candidates: FocusCandidate[]) => FocusCandidate | undefined;
+
+// FocusArbitrator callback will be called with an array of FocusCandidate.
+// See View's arbitrateFocus property.
+export interface FocusCandidate {
+    // An instance of the component which wants to be focused.
+    component: RX.FocusableComponent;
+
+    // Returns component.props.accessibilityId (if specified).
+    accessibilityId?: string;
+
+    // If the candidate is inside the View with arbitrateFocus property specified,
+    // returns accessibilityId of that View.
+    parentAccessibilityId?: string;
```

Comment:
Why would we include this field? This should be the same for all candidates passed to the focus arbitrator. And the caller should already know what the parent accessibility ID is because the focus arbitrator was installed on that node. I recommend removing this.